addon/components/freestyle-palette/index.hbs

```diff
@@ -1,7 +1,7 @@
 <div class='FreestylePalette'>
-  <div class='FreestylePalette-title'>{{@title}}</div>
```

Comment:
Using `this.` for `title` and `description` so we can fall back to the default values.